cmake/templates/OpenVINOConfig.cmake.in

```diff
@@ -283,6 +283,18 @@ if(NOT TARGET openvino)
     set(_ov_as_external_package ON)
     include("${CMAKE_CURRENT_LIST_DIR}/OpenVINOTargets.cmake")
 
+    foreach(target openvino::runtime openvino::runtime::c
+               openvino::frontend::onnx openvino::frontend::paddle openvino::frontend::tensorflow
+               openvino::frontend::pytorch openvino::frontend::tensorflow_lite)
```

Comment:
we also have a list of all targets here:
https://github.com/openvinotoolkit/openvino/blob/master/cmake/templates/OpenVINOConfig.cmake.in#L329-L331
let's declare a list of targets in a single place and reuse in both places.
Later not to forget to `unset` this variable.

Reply:
The list you mentioned does not include pytorch and tensorflow_lite frontends. Do I need to add them here?

Reply:
yes, it's a mistake that those libs are missed

Reply:
done